esx_service/utils/auth_data_test.py

```diff
@@ -0,0 +1,286 @@
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+
+        tenant1.set_name(self.auth_mgr.conn, 'new_tenant1')
+
+    
+    def test_set_description(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        
+        privileges = self.get_privileges()
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+        tenant1.set_description(self.auth_mgr.conn, 'new description')
+    
+    def test_set_default_datastore_and_privileges(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        
+        privileges = self.get_privileges()
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+
+        default_datastore = 'new_default_ds'
+        default_privileges = [{'datastore': default_datastore,
+                              'global_visibility': 0,
+                              'create_volume': 1,
+                              'delete_volume': 1,
+                              'mount_volume': 1,
+                              'max_volume_size': 0,
+                              'usage_quota': 0}]
+        tenant1.set_default_datastore_and_privileges(self.auth_mgr.conn, default_datastore, default_privileges)                              
+
+    
+    def test_add_datastore_access_privileges(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        privileges = []
+        
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+        
+        vms = [(str(uuid.uuid4()), 'vm2')]
+        privileges = []
+        tenant2 = self.auth_mgr.create_tenant('tenant2', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant2.id))
+        
+        privileges = [{'datastore': 'datastore1',
+                       'global_visibility': 0,
+                       'create_volume': 0,
+                       'delete_volume': 0,
+                       'mount_volume': 0,
+                       'max_volume_size': 0,
+                       'usage_quota': 0}]
+
+        tenant1.set_datastore_access_privileges(self.auth_mgr.conn, privileges)
+
```

Comment:
where is comparison with expected results, or check that privileges really honored ? 
You may want to check other tests, they usuallu defined  "params, expected_results" table and then scan over it